docker-scout/Jenkinsfile

```diff
@@ -81,6 +101,10 @@ def dockerScoutContainer() {
   return 'docker/scout-cli:1.4.1'
 }
 
+def axonivyEngineImageUrl(String version) {
+  return 'registry://axonivy/axonivy-engine:' + version
```

Comment:
why you need registry:// here?

Reply:
Then always the lates form the registry is used (not a cached one): https://github.com/docker/scout-cli/blob/main/docs/scout_cves.md#description